src/ApiService/ApiService/onefuzzlib/orm/EntityConverter.cs

```diff
@@ -156,16 +156,20 @@ public class EntityConverter {
         if (type is null) {
             throw new NullReferenceException();
         }
-        var tableEntity = new TableEntity();
+
         var entityInfo = GetEntityInfo<T>();
-        foreach (var prop in entityInfo.properties.SelectMany(x => x)) {
-            //var prop = kvp.First();
+        Dictionary<string, object?> columnValues = entityInfo.properties.SelectMany(x => x).Select(prop => {
             var value = entityInfo.type.GetProperty(prop.name)?.GetValue(typedEntity);
+            if (value == null) {

```

Comment:
this is the actual fix. The null value was converted to the string "null" before this change